tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/MsTestIntegration.cs

```diff
@@ -285,7 +285,9 @@ internal static TestModule GetOrCreateTestModuleFromTestAssemblyInfo<TAsmInfo>(T
                 }
 
                 Common.Log.Debug("Module: {Module}, Framework version: {Version}", assemblyName, frameworkVersion);
-                return TestModule.InternalCreate(assemblyName, CommonTags.TestingFrameworkNameMsTestV2, frameworkVersion);
+                var newModule = TestModule.InternalCreate(assemblyName, CommonTags.TestingFrameworkNameMsTestV2, frameworkVersion);
+                newModule.EnableIpcClient();
```

Comment:
If we _always_ need to enable the IpcClient, is it worth moving this to `InternalCreate` - or is it only required by some frameworks?

Reply:
`Create` calls `InternalCreate` and I don't want to enable the IpcClient when a customer uses the manual Api. That's why everything is internal here and called explicitly.

Reply:
Makes sense, thanks